priv/module_static/lib/runes.js

```diff
@@ -0,0 +1,6463 @@
+(function (exports) {
```

Comment:
same as for binary_tree.js

Reply:
Same as above. Modules files are added to .gitignore.